css-attribute-case-sensitivity/index.html

```diff
@@ -0,0 +1,36 @@
+---
+feature_name: Case-insensitive Attribute Selector Matching
+chrome_version: 49
+feature_id: 5610936115134464
+---
+
+<h3>Background</h3>
+<p>To allow an author to match a CSS attribute (ASCII) case-insensitively, an
+additional modifier (<code>i</code>) for CSS attribute selectors has been
+added.</p>
+<p>By default, case-sensitivity of attribute names and values in selectors
+depends on the document language.</p>
+
+{% capture css %}
+/* All anchors which have a download attribute with the ending `pdf`. */
+a[download$="pdf" i] { 
+  color: green;
+}
+/* All anchors which have a download attribute with the exact ending `pdf` but
+ * not the value endings `PDF` or `pDf`. */
+a[download$="pdf"] { 
+  font-weight: bold;
+}
+{% endcapture %}
+
+{% include css_snippet.html css=css %}
+{% capture html %}
+<p>
+  <a href="#" download="foo.pdf">foo.pdf</a>
```

Comment:
Could you rewrite this example to use an HTML attribute like `name` that doesn't have user-visible side effects, instead of `download`? There's no benefit to triggering downloads of non-existent files whenever someone clicks on one of these links.